overlord/snapstate/storehelpers.go

```diff
@@ -59,16 +59,9 @@ func updateInfo(st *state.State, snapst *SnapState, channel string, userID int)
 
 	theStore := Store(st)
 	st.Unlock() // calls to the store should be done without holding the state lock
-	res, err := theStore.ListRefresh([]*store.RefreshCandidate{refreshCand}, user)
+	res, err := theStore.LookupRefresh(refreshCand, user)
 	st.Lock()
-	if err != nil {
-		return nil, fmt.Errorf("cannot get refresh information for snap %q: %s", curInfo.Name(), err)
```

Comment:
I don't think so; leaving this ended with errors saying things like, for example
```
error: cannot refresh "core": cannot get refresh information for snap "core": cannot query the
       store for updates: got unexpected HTTP status code 404 via POST to
       "http://localhost:9999/snaps/metadata"
```
I don't think the "cannot get refresh information" bit is adding anything, there.